.eslintrc

```diff
@@ -22,6 +28,37 @@
     "prefer-template": 0,
     "no-console": 0,
     "react/prefer-stateless-function": 0,
+    "react/no-unused-prop-types": 0,
+    "react/require-default-props": 0,
+    "react/forbid-prop-types": 0,
+    "react/destructuring-assignment": 0,
+    "object-curly-newline": 0,
+    "prefer-destructuring": 0,
+    "import/extensions": 0,
+    "operator-linebreak": 0,
+    "import/order": 0,
+    "lines-between-class-members": 0,
+    "no-plusplus": 0,
+    "padded-blocks": 0,
+    "import/first": 0,
+    "arrow-parens": 0,
+    "prefer-promise-reject-errors": 0,
+    "react/jsx-filename-extension": 0,
+    "function-paren-newline": 0,
+    "no-else-return": 0,
+    "react/no-string-refs": 0,
+    "react/forbid-foreign-prop-types": 0,
+    "no-multi-spaces": 0,
+    "no-restricted-properties": 0,
+    "class-methods-use-this": 0,
+    "import/no-cycle": 0,
+    "react/sort-comp": 0,
+    "import/no-dynamic-require": 0,
+    "import/named": 0,
+    "camelcase": 0,
+    "import/no-extraneous-dependencies": 0,
+    "react/default-props-match-prop-types": 0,
+    "implicit-arrow-linebreak": 0,
```

Comment:
Is the thinking here that after we merge the new rules we'll fix them separately in subsequent PRs?

Also, can we alpha order these entries? Thank you!

Reply:
That was the thinking.  I did not want to make any changes except for whitespace-related issues in order to guarantee functionality.

Many of these can be fixed with eslint --fix, but I did not want to run that risk due to not having test cases in this repo.

I will clean up the ordering.